runtime/caml/addrmap.h

```diff
@@ -0,0 +1,97 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*                 Stephen Dolan, University of Cambridge                 */
+/*                                                                        */
+/*   Copyright 2015 University of Cambridge                               */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+#include "mlvalues.h"
+
+#ifndef CAML_ADDRMAP_H
+#define CAML_ADDRMAP_H
+
+/* An addrmap is a value -> value hashmap, where
+   the values are blocks */
+
+struct addrmap_entry { value key, value; };
+struct addrmap {
+  struct addrmap_entry* entries;
+  uintnat size;
+};
+
+#define ADDRMAP_INIT {0,0}
+
+int caml_addrmap_contains(struct addrmap* t, value v);
+value caml_addrmap_lookup(struct addrmap* t, value v);
```

Comment:
```suggestion
int caml_addrmap_contains(struct addrmap* t, value k);
value caml_addrmap_lookup(struct addrmap* t, value k);
```

Reply:
Fixed in https://github.com/ocaml/ocaml/pull/10831/commits/47b47d57518cb8b9d832f41f7f8c83f9243e9c03